sips/sip-028/sip-028-sbtc_peg.md

```diff
@@ -0,0 +1,203 @@
+
+sBTC Signers are responsible for accepting or rejecting all sBTC deposit and withdrawal operations submitted to the network. For a transaction to be fulfilled, at least 70% of the signers need to approve the transaction. This means that the liveness and reliability of the signers is crucial to the success of the protocol. The system is live ("resilient") if at least 70% of the sBTC Signer voting power are online and honest. Then (and only then), deposits and withdrawals happen in a timely manner. The system is safe ("trustworthy") if at least 30% of the sBTC Signer voting power is honest. Then, no theft of funds can occur.
+
+Throughout this release, sBTC will have an unchangeable and distinct signer set that will not explicitly be part of Stacks consensus. Given that there will be 15 unique signers that shall comprise the system, and each unique signer is allocated exactly one vote, it would require at least 11 out of 15 signatures for an sBTC operation to be fulfilled.
+
+While up to 30% of the signers can be offline without a user impact on the functioning of the protocol, it becomes more critical for the rest of the signers to approve sBTC operations because operations necessarily still need to meet 70% of the original signing power. If more than 30% of signers become unavailable, no sBTC operations will be approved because it will be impossible to get 70% approval when less than 70% are online. An operation that isn’t approved will become reclaimable by the user after a timeout has elapsed. The timeout is specified by the user when preparing the deposit request and is measured in Bitcoin blocks.
+
+### sBTC Signer Responsibilities
+
+Overview of tasks the sBTC signers carry out:
+- Accept requests to deposit BTC.
+- Fulfill requests to withdraw BTC.
+- Complete deposit and withdrawal requests in a timely manner.
+- Maintain industry standard operational security around any hosts and private data (to include private keys).
+- Move BTC to a new UTXO when private keys are rotated.
+- Signers must perform UTXO consolidate as it is deposited [1].
+- Signers must deduct transaction fees from users in order to fund BTC withdrawal transactions:
+  - Ensure that the transaction fee is paid for (e.g., they deduct it from the user, and they set a minimum sBTC withdrawal amount).
+  - Transaction fee must be estimated proportionally for the requested operation [2].
+- Collectively, the signers must coordinate to calculate and advertise the fee parameters of the system:
+  - A minimum sBTC peg-out.
+  - STX transaction fee for minting the sBTC. This fee is paid by the user and can be sponsored by a 3rd party, which is described in the [Auxiliary Features](#auxiliary-features) section of this document.
```

Comment:
Auxiliary Features section which is referenced here is missing. Please add this section or remove the reference

Reply:
removed